app/grandchallenge/reader_studies/views.py

```diff
@@ -166,6 +166,28 @@ def delete(self, request, *args, **kwargs):
         return super().delete(request, *args, **kwargs)
 
 
+class ReaderStudyLeaderBoard(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, DetailView
+):
+    model = ReaderStudy
+    permission_required = (
+        f"{ReaderStudy._meta.app_label}.view_{ReaderStudy._meta.model_name}"
```

Comment:
Do we want all of the readers of the study to see this? Won't they see everyone elses scores?

Reply:
I was unsure about that. Should we just make this view available to editors and only show the reader's own score somewhere on the reader study page?

Reply:
Lets go with editors only for now

Reply:
Changed.